modules/administration-guide/pages/configuring-che-storage-size.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: CONCEPT
+:description: Configuring the storage size in {prod-short}
+:keywords: administration guide, configuring, {prod-id-short}, storage size, PVC size, pvc
+:navtitle: Configuring the storage size in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-size.adoc
+
+[id="configuring-{prod-id-short}-workspace-pvc-size"]
+= Configuring the storage size in {prod-short}
```

Comment:
The `id` and the `=` title have to be the same (except for the lowercase dashed syntax).
I'm happy to offer a suggestion, but what I don't current understand is what precisely the storage size is for. Is it storage size per workspace? For example:
```suggestion
[id="configuring-workspace-storage-size"]
= Configuring workspace storage size
```